include/xcfun.h

```diff
@@ -23,9 +23,9 @@ extern "C" {
 enum xc_mode
 {
```

Comment:
Could you rewrite the `enum` comments in this style:
```
/*! \enum pcmsolver_reader_t
 *  \brief Input processing strategies
 */
typedef enum {
  PCMSOLVER_READER_OWN, /*!< Module reads input on its own */
  PCMSOLVER_READER_HOST /*!< Module receives input from host */
} pcmsolver_reader_t;     
```
and give a brief documentation of the `enum`-s too?